sites/public/cypress/e2e/account/account.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { publicUser, updatedPublicUser } from "../../mockData/userData"
```

Comment:
We can't test changing the email here because it requires email verification, but there is this issue I logged with the email flow: https://github.com/bloom-housing/bloom/issues/4122